packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1879,9 +1890,9 @@ abstract class BuildContext {
   /// This is typically called implicitly from `of()` static methods, e.g.
   /// [Theme.of].
   ///
-  /// This should not be called from widget constructors or from
+  /// This method should not be called from widget constructors or from
   /// [State.initState] methods, because those methods would not get called
-  /// again if the inherited value were to change. To ensure that the widget
+  /// again if the inherited value was to change. To ensure that the widget
```

Comment:
why "was"?

Reply:
Not sure what I were thinking here. Reverted.